docs/reference/settings.rst

```diff
@@ -41,6 +41,20 @@ When ``WAGTAIL_APPEND_SLASH`` is ``False``, requests to Wagtail pages will be se
 
 .. _this Google Search Central Blog post: https://developers.google.com/search/blog/2010/04/to-slash-or-not-to-slash
 
+ADMIN BASE URL
+==============
+
+``WAGTAILADMIN_BASE_URL``
+--------------------------
+
+.. code-block:: python
+
+    WAGTAILADMIN_BASE_URL = 'http://example.com/'
+
+This is the base URL used by the Wagtail admin site. It is typically used for generating URLs to include in emails.This setting was previously named 'BASE_URL'.
```

Comment:
```suggestion
This is the base URL used by the Wagtail admin site. It is typically used for generating URLs to include in notification emails.
```
Remove the previous name here, we will add that below (see other comment), plus a small fix for the usage note.